plugins/importexport/native/NativeImportExportDeployment.inc.php

```diff
@@ -59,6 +59,22 @@ function getRepresentationNodeName() {
 	function getSchemaFilename() {
 		return 'native.xsd';
 	}
+
+	/**
+	 * Get the publication node name
+	 * @return string
+	 */
+	function getPublicationNodeName() {
```

Comment:
I don't think this abstraction adds anything, since we're using `publication` (and `publications`) throughout -- I think this pattern comes from cases where we need different element names (`article` in OJS, `monograph` in OMP). So I'd suggest removing it.

Reply:
[removed from OJS and pkp-lib, can remove here without anything more]

Reply:
Removed